Lesson_09_fix/l09_h01_read_write_files.py

```diff
@@ -0,0 +1,35 @@
+"""
+HW:9 fix[Volkov Anton].
+
+write script that will read "input.txt" file and find there all characters
+from english alphabet
+collect these letter characters and their positions in file
+after info collected this info as text
+write to "output.txt" file in such format
+####################
+<first found letter> -> pos1
+<next_letter> -> pos2
+<next_letter -> pos3
+etc
+#######################
+"""
+import re
+
+SEPARATOR = '#' * 15 + '\n'
+add_print = ''
+positions = []
+# Read from files alphabet letter and their positions
+with open('input.txt', 'r') as fh:
+    text = fh.read()
+    for match in re.finditer(r'(?<![A-Za-z])[A-Za-z](?![A-Za-z])', text):
+        letter = match.group()
+        position = match.start()
+        positions.append((letter, position))
+    for letter, pos in positions:
```

Comment:
This part better to do during write to file operation. No sense keep it here during file read